Sources/Controllers/2.0/OAEditTargetViewController.mm

```diff
@@ -798,6 +801,15 @@ - (void) tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
         }
         case 5: // coords
         {
+            UITableViewCell *cell = [_tableView cellForRowAtIndexPath:indexPath];
+            if ([cell isKindOfClass:OATargetInfoCollapsableCoordinatesViewCell.class])
+            {
+                self.collapsableCoordinatesView.collapsed = !self.collapsableCoordinatesView.collapsed;
```

Comment:
Why collapsableCoordinatesView not in OATargetInfoCollapsableCoordinatesViewCell?